controllers/login.js

```diff
@@ -0,0 +1,25 @@
+const loginService = require("../services/login")
+const errors = require("../utils/errors")
+
+
+const loginUser = async (req, res) => {
+    try {
+        const { email, password } = { ...req.body }
+
+        const { userId, token } = await loginService.loginUser(email, password)
+
+        return res.json({
```

Comment:
I think you are missing status code here.

Reply:
It defaults to 200, but I will explicitly write it